projects/packages/masterbar/src/admin-menu/load.php

```diff
@@ -48,8 +48,7 @@ function get_admin_menu_class() {
 		$is_difm_lite_in_progress = wpcomsh_is_site_sticker_active( 'difm-lite-in-progress' );
 		$is_support_session       = defined( 'WPCOM_SUPPORT_SESSION' ) && WPCOM_SUPPORT_SESSION;
 		if ( $is_difm_lite_in_progress && ! $is_support_session ) {
-			require_once __DIR__ . '/class-domain-only-admin-menu.php';
-			return Domain_Only_Admin_Menu::class;
+			return '';
```

Comment:
We should be able to remove this once we get rid of all the Admin Menu classes. If we don't return an empty string here, DIFM sites would get the `WPCom_Admin_Menu` class and suddenly display a sidebar upsell.